modules/related-posts/jetpack-related-posts.php

```diff
@@ -796,7 +796,7 @@ public function get_for_post_id( $post_id, array $args ) {
 			$options['size'] = $args['size'];
 		}
 
-		if ( 0 === (int) $post_id || empty( $options['size'] ) ) {
```

Comment:
Can we break it down to a few lines so that it is easier to understand. 

Maybe something like:
```
if (
 ! $options['enabled'] 
|| 0 === (int)$post_id 
|| empty( $options['size'] ) 
|| ( $post_id && get_post_status( $post_id ) !== 'publish' )
) {
```

Reply:
Addressed in 5fdc6d700ec558dd644b44b1588b48a07462db37